example/example.php

```diff
@@ -2,7 +2,7 @@
 
 use jblond\Diff;
 use jblond\Diff\Renderer\Html\Inline;
-use jblond\Diff\Renderer\Html\SideBySide;
+use jblond\Diff\Renderer\Html\Merged;use jblond\Diff\Renderer\Html\SideBySide;
```

Comment:
Please add a line break

Reply:
Sorry about that. It seems the IDE doesn't properly format the code when it has mixed languages.